share/completions/pbpaste.fish

```diff
@@ -0,0 +1,3 @@
+complete -c pbpaste -s help -d "Display command help"
+complete -c pbpaste -s pboard -d "Specifies which paste specifies which pasteboard to copy to or paste from. If no pasteboard is given, the general pasteboard will be used by default. {general | ruler | find | font}"
```

Comment:
Is "{general | ruler | find | font}" the list of possible pboards?

If so, you can just add them as arguments:

```fish
complete -c pbpaste -s pboard -d 'Which pasteboard to use' -xa 'general ruler find font'
```

Also these descriptions are pretty long, we try to keep it short.

Reply:
Oh also `-s pboard` is invalid - `-s` introduces a *short* option, meaning it is `-X`, with one character after the dash, and can be grouped etc (like in `ls -lah`).

If this is `pbpaste --pboard`, use `-l`, if it is `pbpaste -pboard` (one dash), use `-o`.